util/topgen/lib.py

```diff
@@ -365,3 +406,15 @@ def get_base_and_size(name_to_block: Dict[str, IpBlock],
         assert isinstance(base_addr, int)
 
     return (base_addr, size_byte)
+
+
+def get_io_enum_literal(sig, prefix):
+    """Returns the DIO pin enum literal with value assignment"""
+    if sig['idx'] < 0:
```

Comment:
Can you add a comment explaining this special case?

Reply:
Done